components/builder-api/src/server/resources/settings.rs

```diff
@@ -47,7 +50,9 @@ impl Settings {
            .route("/settings/{origin}/{name}",
                   web::get().to(get_origin_package_settings))
            .route("/settings/{origin}/{name}",
-                  web::put().to(update_origin_package_settings));
+                  web::put().to(update_origin_package_settings))
+           .route("/settings/{origin}/{name}",
+                  web::delete().to(delete_origin_package_settings));
```

Comment:
Splendid! Can we get an entry into our raml doc for this route as well?

Reply:
Absolutely, good catch!